modules/apps/headless/headless-admin-user/headless-admin-user-test/src/testIntegration/java/com/liferay/headless/admin/user/resource/v1_0/test/AccountResourceTest.java

```diff
@@ -388,6 +438,40 @@ public void testPostOrganizationAccountsByExternalReferenceCode()
 		}
 	}
 
+	@Override
+	@Test
+	public void testPostOrganizationByExternalReferenceCodeAccountsByExternalReferenceCode()
```

Comment:
why we cannot use the default test in the base class?

Reply:
@smottal same explanation as above.